hawkbit-sdk/hawkbit-sdk-dmf/src/main/java/org/eclipse/hawkbit/sdk/dmf/DmfController.java

```diff
@@ -83,10 +85,24 @@ public void stop() {
     }
 
     public void processUpdate(final EventTopic actionType, final DmfDownloadAndUpdateRequest updateRequest) {
-        updateHandler.getUpdateProcessor(this, actionType, updateRequest);
+        log.info("Processing update for {} actionId : {}", controllerId, updateRequest.getActionId());
```

Comment:
For logging  of controller bound events alsways use pattern:
log.info(LOG_PREFIX + ""Processing update for {} actionId : {}"", getTenantId(), getControllerId(), controllerId, updateRequest.getActionId())
(as for instance in UpdateHandler (constant LOG_PREFIX could be made shared))

Reply:
fixed